node/init_test.go

```diff
@@ -0,0 +1,17 @@
+package node
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestInit(t *testing.T) {
```

Comment:
Proposal: Can we have a dedicated issue for adding more tests for node init?

Reply:
Yes. Bringing the suggestion to maintain a list of missing tests in here.

Reply:
```suggestion
// TODO: We need more test coverage for Init part
// Tests could include invalid paths/configs and custom ones
// For more info visit #89 
func TestInit(t *testing.T) {
```

Reply:
@Bidon15, pls update your suggestion with the one who left it:
// TODO(@Bidon15): ...

Reply:
```suggestion
// TODO(@Bidon15): We need more test coverage for Init part
// Tests could include invalid paths/configs and custom ones
// For more info visit #89 
func TestInit(t *testing.T) {
```